frontend/public/components/edit-yaml.jsx

```diff
@@ -293,7 +293,7 @@ export const EditYAML = connect(stateToProps)(
                       <span className="pficon pficon-info"></span>This object has been updated. Click reload to see the new version.
                     </p>}
                     {create && <button type="submit" className="btn btn-primary" id="save-changes" onClick={() => this.save()}>Create</button>}
-                    {!create && <button type="submit" className="btn btn-primary" id="save-changes" onClick={() => this.save()}>Save Changes</button>}
+                    {!create && !readOnly && <button type="submit" className="btn btn-primary" id="save-changes" onClick={() => this.save()}>Save Changes</button>}
```

Comment:
I'd think we'd want to make the editor read only too instead of just removing the button.

Reply:
Ok, made the yaml editor read only.  Also removed the Cancel button for read only since it seemed to indicate you could cancel changes made to the yaml:
![image](https://user-images.githubusercontent.com/12733153/44038764-83cc3796-9ee5-11e8-833a-00c6a9f7a49e.png)
@tlwu2013 Wondering if we should change the background color of a 'read only' editor to perhaps something light grey to indicate disabled?  At the moment it looks the same as an editable editor.

Reply:
>  Also removed the Cancel button for read only since it seemed to indicate you could cancel changes made to the yaml

There are some links that take you directly to the YAML page, then cancel takes you back. This still might be useful even if readonly.

Reply:
Ok, restored Cancel button for readOnly